jib-core/src/main/java/com/google/cloud/tools/jib/image/json/JsonToImageTranslator.java

```diff
@@ -196,5 +202,25 @@
     return ports.build();
   }
 
+  /**
+   * Converts a map of volumes strings to a list of {@link AbsoluteUnixPath}s * (e.g. {@code {@code
```

Comment:
Looks like there's an extra `*` in this line.

Reply:
removed.